PlayCover/Model/PlayApp.swift

```diff
@@ -82,6 +82,12 @@ class PlayApp: BaseApp {
 
     func runAppExec() {
         let config = NSWorkspace.OpenConfiguration()
+        // This is to prevent Xcode from attaching debugging-related variables
+        // so that they are no longer inherited by the child process.
+        // which fail to load inside iOS apps (missing symbols like _OBJC_CLASS_$_AVPlayerView).
+        for (key, _) in ProcessInfo.processInfo.environment where key.hasPrefix("DYLD_") {
```

Comment:
Will this also remove `DYLD_LIBRARY_PATH`, which `PlayApp` inserts for `Introspection` and `iOSFramework` on line `221`?

Reply:
No, unsetenv only remove DYLD_* variables that exist in PlayCover’s env process when the game is launched. DYLD_LIBRARY_PATH is being injected on runtime via info.plist iirc